data/json/itemgroups/guns.json

```diff
@@ -200,11 +201,12 @@
       [ "ak47", 100 ],
       [ "ak74", 60 ],
       [ "an94", 40 ],
+      [ "arx160", 40 ],
+      [ "cane_rifle", 80 ],
```

Comment:
It doesn't fit logically with the other items in that grouping. If there are changes you'd like to propose to other guns a separate PR would be the best route.